commons-db/src/main/java/com/palantir/nexus/db/sql/BasicSQLUtils.java

```diff
@@ -343,6 +344,19 @@ public static void toStringSqlArgs(final StringBuilder sb, Object[] args) {
         }
     }
 
+    private static final int MAX_ARRAY_PRINT_BYTES = 64;
+
+    // Oracle use HEXTORAW('0f1f') to create a 2 byte RAW literal
+    // Postgres use '\\x0f1f' to create a 2 byte bytea literal
+    private static void prettyPrintByteArray(final StringBuilder sb, byte[] args) {
+        for (int i = 0; i < Math.min(args.length, MAX_ARRAY_PRINT_BYTES); i++) {
```

Comment:
minor nit that you might want to pull out the Math.min so it doesn't repeatedly call min

```
    int printLength = Math.min(args.length, MAX_ARRAY_PRINT_BYTES);
    for (int i = 0; i < printLength; i++) {
```